src/main/java/org/geomerty/oop/shapes/Circle.java

```diff
@@ -0,0 +1,21 @@
+package org.geomerty.oop.shapes;
+
+import static java.lang.Math.PI;
+
+public class Circle implements iShape {
+    private final double radius;
+
+    public Circle(double radius) {
+        this.radius = radius;
+    }
+
+    @Override
+    public void area() {
+        System.out.println("Area of circle is " + PI * radius * radius);
+    }
+
+    @Override
+    public void perimeter() {
+        System.out.println("Perimeter of circle is " + 2 * PI * radius);
```

Comment:
@LouiseLM I would keep calculation separate from the representation (printing to the screen). It could be done by splitting methods to two: `public double perimeter()` and `public void printPerimeter()`. Actually printing methods could be moved to a separate class altogether.  It makes code more testable and reusable.